hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocol/ScmBlockLocationProtocol.java

```diff
@@ -138,4 +139,6 @@ List<AllocatedBlock> allocateBlock(long size, int numBlocks,
    */
   List<DatanodeDetails> sortDatanodes(List<String> nodes,
       String clientMachine) throws IOException;
+
+  InnerNode getClusterTree() throws IOException;
```

Comment:
Could you add a javadoc for this new API?

Reply:
getClusterTree -> getNetworkTopology

Reply:
Thank you @ChenSammi, I have added a javadoc and renamed the existing method.